howto/deploy-k8s-prod/README.md

```diff
@@ -0,0 +1,186 @@
+dapr-dashboard-dcd7cc8fb-ml2p8           1/1     Running   0          12s
+dapr-operator-7b57d884cd-6spcq           1/1     Running   0          12s
+dapr-placement-86b76f6545-vkc6f          1/1     Running   0          12s
+dapr-sentry-7c7bf75d8b-dcnhm             1/1     Running   0          12s
+dapr-sidecar-injector-7b847db96f-kwst9   1/1     Running   0          12s
+```
+
+You can verify the health and version of the control plane using the Dapr CLI:
+
+```bash
+dapr status -k
+
+NAME                   NAMESPACE      HEALTHY  STATUS   VERSION  AGE  CREATED
+dapr-placement         dapr-system    True     Running  0.9.0    12s  2020-07-24 15:38.15
+dapr-operator          dapr-system    True     Running  0.9.0    12s  2020-07-24 15:38.15
+dapr-sidecar-injector  dapr-system    True     Running  0.9.0    12s  2020-07-24 15:38.16
+dapr-sentry            dapr-system    True     Running  0.9.0    12s  2020-07-24 15:38.15
+```
+
+*Note: If new fields have been added to the target Helm Chart being upgraded to, the `helm upgrade` command will fail. If that happens, you need to find which new fields have been added in the new chart and add them as parameters to the upgrade command, for example: `--set <field-name>=<value>`.*
+
+#### Updating the data plane (sidecars)
+
+The last step is to update pods that are running Dapr to pick up the new version of the Dapr runtime.
+To do that, simply issue a rollout restart command for any deployment that has the `dapr.io/enabled` annotation:
+
+```
+kubectl rollout restart deploy/<DEPLOYMENT-NAME>
+```
+
+## Recommended security configuration
+
+Properly configured, Dapr not only be secured with regards to it's control plane and sidecars communication, but can also make your application more secure with a number of built-in features.
+
+It is recommended that a production-ready deployment includes the following settings:
+
+1. Mutual Authentication (mTLS) should be enabled. Note that Dapr has mTLS on by default. For details on how to bring your own certificates, see [here](../configure-mtls/README.md#bringing-your-own-certificates)
+
+2. Dapr API authentication is enabled (this is the between your application and the Dapr sidecar). To secure the Dapr API from unauthorized access, it is recommended to enable Dapr's token based auth. See [here](../enable-dapr-api-token-based-authentication/README.md) for details
+
+3. All component YAMLs should have secret data configured in a secret store and not hard-coded in the YAML file. See [here](../../concepts/secrets/component-secrets.md) on how to use secrets with Dapr components
+
+4. The Dapr control plane is installed on a separate namespace such as `dapr-system`, and never into the `default` namespace
+
+Dapr also supports scoping components for certain applications. This is not a required practice, and can be enabled according to your Sec-Ops needs. See [here](../components-scopes/README.md) for more info.
+
+## Tracing and metrics configuration
+
+TBD
```

Comment:
Can you can add something here? I presume this is 1) Install Promethues 2) Configure tracing on per component basis 3)??

Reply:
Added.